packages/next-on-pages/src/buildApplication/generateGlobalJs.ts

```diff
@@ -20,6 +20,8 @@ export function generateGlobalJs(): string {
 				env: new Proxy(
 					{},
 					{
+						ownKeys: () => Reflect.ownKeys(envAsyncLocalStorage.getStore()),
+						getOwnPropertyDescriptor: () => ({ enumerable: true, configurable: true }),
```

Comment:
I don't think that this is correct as it would return non-`undefined` for non existing properties, it also doesn't return the `value` of the property when it should, and in general this simply hardcodes a result for all properties

what about?
```suggestion
            getOwnPropertyDescriptor: (_, ...args) => Object.getOwnPropertyDescriptor(als.getStore(), ...args),
```

it does seem to work as expected: https://stackblitz.com/edit/stackblitz-starters-a2p5yp?file=index.mjs

Reply:
fine by me, i just found this from stack overflow 👍🏻